docs/source/faq.rst

```diff
@@ -55,7 +55,7 @@ FAQ
 
 * What is a rollup?
 
-  A rollup performs transaction executioon oustide Layer 1 nad then the data is
+  A rollup performs transaction executioon oustide Layer 1 and then the data is
   posted to layer 1 where consensus is reached. There are two types of rollups,
   Zero-Knowledge (ZK) and Optimistic Rollups.
```

Comment:
It would be nice to also fix consistency when referencing "layer 1", while we're here.

Reply:
What do you mean with fix consistency when referencing "layer 1"? That we go for either "Layer 1" or "L1"???

Reply:
We have "Layer 1" and then immediately below "layer 1" -- note the case change.